eng/mgmt/automation/generation.yml

```diff
@@ -0,0 +1,61 @@
+trigger: none
+
+pr: none
+
+pool:
+  vmImage: 'ubuntu-18.04'
+
+variables:
+  MAVEN_CACHE_FOLDER: $(Pipeline.Workspace)/.m2/repository
+  MAVEN_OPTS: '-Dmaven.repo.local=$(MAVEN_CACHE_FOLDER)'
+  NodeVersion: '14.x'
+
+steps:
+- task: Cache@2
+  inputs:
+    key: 'maven | "$(Agent.OS)" | **/pom.xml'
+    restoreKeys: |
+      maven | "$(Agent.OS)"
+      maven
```

Comment:
This looks a little off. Why would you need maven in the key twice?

Reply:
This is just copied from azure devops docs https://docs.microsoft.com/en-us/azure/devops/pipelines/release/caching?view=azure-devops#maven